collectors/python.d.plugin/bigquery/README.md

```diff
@@ -0,0 +1,62 @@
+<!--
+title: "BigQuery"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/bigquery/README.md
+-->
+
+# BigQuery Netdata Collector
+
+A python based collector leveraging [`pandas-gbq`](https://pandas-gbq.readthedocs.io/en/latest/) to run a query 
+against [BigQuery](https://cloud.google.com/bigquery) and save results into a chart in Netdata.
+
+## Requirements
+
+This collector depends on some Python packages that can usually be installed via `pip`.
```

Comment:
Nowadays we still have have `pip` and `pip3` on hosts, my initial understanding, and probably I am wrong, this collector needs `python 2`. @Ferroin am I missing something with these `pip` versions?

Reply:
Looks like `pandas-gbq` is only python 3. https://pypi.org/project/pandas-gbq/

So i'll need to add something maybe to make this clearer.

Reply:
I think so, only to be sure that users won't have the confusion I had.